src/lib/layer-manager.js

```diff
@@ -78,21 +82,33 @@ export default class LayerManager {
       shaderCache: new ShaderCache({gl})
     };
 
+    seerInitListener(this._initSeer);
+    layerEditListener(this._editSeer);
+
     Object.seal(this.context);
     Object.seal(this);
+  }
 
-    seerInitListener(() => {
-      this.layers.forEach(layer => {
-        initLayerInSeer(layer);
-        updateLayerInSeer(layer);
-      });
+  _initSeer() {
```

Comment:
Nit: One line comment header would be an opportunity to explain e.g. when this is called and why